06_performance_monitor/performance_monitor_spec.rb

```diff
@@ -33,8 +33,10 @@
   it "takes about 1 second to run a block that sleeps for 1 second" do
     elapsed_time = measure do
       sleep 1
+      puts "here"
```

Comment:
This does not need to be here. Debug statements should not be present in final PRs